vpc/basic_firewall_rule/main.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2022 Google LLC
```

Comment:
If this is a new sample, please update year to 2023.

Reply:
This is an old file from 2022 as mentioned at https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/firewall_with_target_tags/main.tf

Reply:
Make sense.